packages/config/src/json/json.config.ts

```diff
@@ -171,6 +171,7 @@ export class ConfigJson {
     // Map the configuration sources into imagery ids
     for (const l of ts.layers) {
       const layer = { ...l };
+      if (layer.disable) layer.minZoom = 32;
```

Comment:
We should support  `disabled` in the `ConfigImagery` and shouldn't need to convert this back to `minZoom` we then just need a check for `disabled` in the tile rendering pipeline.

Reply:
Good point, I thought about this before. However, I believe this `disabled` flag is just for the `ConfigTileSet.ConfigLayer` only. I think the `ConfigImagery` for mainly used for individual imagery and also don't have any zoom level settings? The other one is Virtual `ConfigTileSet` which used for individual as well and always set as minZoom:0 and maxZoom:32.

Reply:
I have removed the set `minzoom:32` and filter out the layers have disabled flag in the api.